dracut/repartition/endless-repartition.sh

```diff
@@ -144,12 +144,21 @@ if [ -n "$swap_start" ]; then
 start=$swap_start, type=0657FD6D-A4AB-43C4-84E5-0933C84B4F4F"
 fi
 
+# Randomize UUIDs
+parts=$(echo "$parts" | while read line; do echo $line | sed -e "s/uuid=.*/uuid=$(uuidgen)/"; done)
```

Comment:
To avoid adding uuidgen to the initramfs, try simply deleting the uuid= part of the line. Hopefully that will cause sfdisk to generate new ones itself. Same for disk label UUID